proto/cosmos/mint/module/v1/module.proto

```diff
@@ -7,7 +7,7 @@ import "cosmos/app/v1alpha1/module.proto";
 // Module is the config object of the mint module.
 message Module {
   option (cosmos.app.v1alpha1.module) = {
-    go_import: "github.com/cosmos/cosmos-sdk/x/mint"
+    go_import: "cosmossdk.io/x/mint"
```

Comment:
missing cosmos/vanity PR

Reply:
https://github.com/cosmos/vanity/pull/52 made the pr